src/Open3D/GUI/FileDialog.cpp

```diff
@@ -121,58 +120,59 @@ class DirEntry {
 };
 
 struct FileDialog::Impl {
-    Mode mode;
-    std::vector<DirEntry> entries;
-    std::shared_ptr<TextEdit> filename;
-    std::shared_ptr<Combobox> dirtree;
-    std::shared_ptr<ListView> filelist;
-    std::shared_ptr<Combobox> filter;
-    std::unordered_map<int, std::unordered_set<std::string>> filterIdx2filter;
-    std::shared_ptr<Horiz> filterRow;
-    std::shared_ptr<Button> ok;
-    std::shared_ptr<Button> cancel;
-    std::function<void()> onCancel;
-    std::function<void(const char *)> onDone;
+    Mode mode_;
+    std::vector<DirEntry> entries_;
+    std::shared_ptr<TextEdit> filename_;
+    std::shared_ptr<Combobox> dirtree_;
+    std::shared_ptr<ListView> filelist_;
+    std::shared_ptr<Combobox> filter_;
+    std::unordered_map<int, std::unordered_set<std::string>>
```

Comment:
I just noticed that filter_idx_2_filter "unordered_map" is used exactly like a vector.  Consider changing it to std::vector in the future?

Reply:
Oh, it is... It must have morphed from something else.